releases/v0.6.0.md

```diff
@@ -0,0 +1,65 @@
+# Release Notes for v0.6.0
+Release Date: June 7th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
+- Attestation Service upgraded to avoid several security issues in Go packages.
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV(-ES)
+- Intel SGX
+
+The following platforms are untested or partially supported:
+- IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+- AMD SEV-SNP
+
+The following platforms are in development:
+- ARM CCA
+
+## Limitations
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * Image signature validation with AMD SEV-ES is not covered by CI.
+- SELinux is not supported on the host and must be set to permissive if in use.
+- Attestation and key brokering support varies by platform.
+  * The generic KBS is only supported on TDX. Other platforms have different solutions.
+- The format of encrypted container images is still subject to change
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Image repository support for encrypted images is unequal
+- CoCo currently requires a custom build of `containerd`
+  * The CoCo operator will deploy the correct version of `containerd` for you
+  * Changes are required to delegate `PullImage` to the agent in the virtual machine
+  * The required changes are not part of the vanilla `containerd`
+  * The final form of the required changes in `containerd` is expected to be different
+  * `crio` is not supported
```

Comment:
```suggestion
`crio` is supported only with `cloud-api-adaptor`

Reply:
Yes would be a fairer way to relate to crio.